app/src/main/java/org/mozilla/focus/activity/EditBookmarkActivity.kt

```diff
@@ -27,12 +27,41 @@ class EditBookmarkActivity : AppCompatActivity() {
     private val bookmark: TempInMemoryBookmarkRepository.Bookmark by lazy { TempInMemoryBookmarkRepository.getInstance().get(itemId) }
     private val name: EditText by lazy { findViewById<EditText>(R.id.bookmark_name) }
     private val location: EditText by lazy { findViewById<EditText>(R.id.bookmark_location) }
-    private val originalName: String by lazy { name.text.toString() }
-    private val originalLocation: String by lazy { location.text.toString() }
+    private val originalName: String by lazy { bookmark.name }
+    private val originalLocation: String by lazy { bookmark.address }
     private lateinit var save: MenuItem
```

Comment:
Similar to the previous comment, the name "save" is quite ambiguous and it's hard to tell what one can do to this variable simply by reading its name.